src/store/flow/flow.ts

```diff
@@ -406,6 +406,41 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
     })
   },
 
+  flow_addMissingResourceValues({getters, rootGetters, dispatch}): void {
```

Comment:
@object-required , any idea about why your [change](https://github.com/FLOIP/flow-builder/pull/335/files#diff-eea673c657b7edcdbf00ab176fcc8ec0a84d21e8a8dfa9ff8a014c7510a144a0R358) is not in `master` anymore